index.js

```diff
@@ -32,7 +32,9 @@ const rehypeReact = require('rehype-react');
  */
 const BaseUrlContext = require('./contexts/BaseUrl');
 
-const Variable = require('@readme/variable');
+const Variable = require('./variable');
+
+export const SelectedAppContext = Variable.SelectedAppContext;
```

Comment:
Can you also retain the `Variable` and `VariablesContext` exports that `@readme/variable` has? We use those in `@readme/syntax-highlighter`: https://github.com/readmeio/syntax-highlighter/blob/main/src/codeMirror/index.jsx#L4

Though I guess that library can load them with `@readme/markdown/variable`? Up to you.

Reply:
Oh, oops. I meant to do that.